extensions/exchanges/gdax/exchange.go

```diff
@@ -12,6 +12,7 @@ var (
 	_ model.Exchange = &gdaxExchange{}
 )
 
+// gdaxExchange implements the representation of a coinbase(gdax) exchange
```

Comment:
Not sure if we really need to have documentation on private level structs. I think that we can remove this comment